### ISSUE TITLE
Sales Force Web to Lead in not working

### ISSUE THREAD

**Issue by Dranetz** [katzwebservices/Gravity-Forms-Salesforce#294]
Last Wednesday (March 23), the Gravity forms Web-to-Lead stopped working. It had been working for us from late January when we installed Sales Force and Gravity forms. The lead goes to Sales Force when we create an HTML form within Sales Force and place it on our web site.
The Gravity Forms are still working, we get the information from the forms in our email. However, the forms do not end up either "talking" to each other, or transferring from the web-to-lead form to Sales Force.
Gravity Forms Version
1.9.17.5
WordPress Version
4.4.1

**Comment by somethumb** [comment]
Yes, this is also happening to me. It's no longer working.

**Comment by thinkfirst29** [comment]
This is also happening with us. it is showing " Success: Your Org ID. is properly configured."

**Comment by klanger** [comment]
Yes, I've got the same weird problem, but I've installed the logging plugin for gravity forms and it tells me that salesforce is responding as if it were accepting the web 2 lead data. It's really weird.

**Comment by klanger** [comment]
I've worked out what went wrong for me. Just disabled the "Require TLS 1.1" critical update in salesforce and I'm ok now. What a mess, with salesforce giving no notification of error whatsoever, just continuing like normal. I've now got a crap ton of data entry to do to catch up.
